openmc/mgxs/mgxs.py

```diff
@@ -2320,24 +2412,45 @@ def __init__(self, domain=None, domain_type=None,
 
 
 class CaptureXS(MGXS):
-    """A capture multi-group cross section.
+    r"""A capture multi-group cross section.
 
     The neutron capture reaction rate is defined as the difference between
     OpenMC's 'absorption' and 'fission' reaction rate score types. This includes
     not only radiative capture, but all forms of neutron disappearance aside
-    from fission (e.g., MT > 100).
+    from fission (i.e., MT > 100).
 
     This class can be used for both OpenMC input generation and tally data
     post-processing to compute spatially-homogenized and energy-integrated
-    multi-group cross sections for multi-group neutronics calculations.
+    multi-group capture cross sections for multi-group neutronics
+    calculations. At a minimum, one needs to set the
+    :attr:`CaptureXS.energy_groups` and :attr:`CaptureXS.domain`
```

Comment:
Yeah, I looked into this for a while actually. Unfortunately it doesn't appear to be possible to have hyperlinks to attribute descriptions.